apps/condo/domains/notification/schema/DisconnectUserFromDeviceService.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Generated by `createservice notification.DisconnectUserFromDeviceService --type mutations`
+ */
+const { GQLCustomSchema } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/notification/access/DisconnectUserFromDeviceService')
+const { Device: DeviceAPI } = require('@condo/domains/notification/utils/serverSchema')
+
+const DisconnectUserFromDeviceService = new GQLCustomSchema('DisconnectUserFromDeviceService', {
+    types: [
+        {
+            access: true,
+            type: 'input DisconnectUserFromDeviceInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, pushTransport: PushTransportType! }',
```

Comment:
Should PushTransportType also be declared here? or is it enough for keystone to generate type from other source?

Reply:
it's declared within `SyncDeviceService`and is working